SmartDeviceLink-iOS/SmartDeviceLink/SDLRPCRequestFactory.h

```diff
@@ -143,6 +147,10 @@
 + (SDLRegisterAppInterface *)buildRegisterAppInterfaceWithAppName:(NSString *)appName languageDesired:(SDLLanguage *)laguageDesired appID:(NSString *)appID;
 //*****
 
+//***** Device Info ******
++(SDLDeviceInfo*) buildDeviceInfo;
```

Comment:
This isn't an RPC request that a developer would ever need access to right? It's actually a struct, and it's only built as part of Register App Interface?

If that's the case, then this can be a private method, which should be prefixed by `sdl_` in order to make it clear what is a private method and what isn't. so it would be `sdl_buildDeviceInfo`. If that's not the case, and the developer does need access to build this struct, what is the use case for that?